src/sage/arith/misc.py

```diff
@@ -2656,6 +2656,11 @@ def factor(n, proof=None, int_=False, algorithm='pari', verbose=0, **kwds):
         Traceback (most recent call last):
         ...
         TypeError: unable to factor 'xyz'
+
+    test that :issue:`35219` is fixed::
```

Comment:
"Test" should start with an uppercase letter here.

Reply:
(Aside from this single-bit detail, it looks good to me.)

Reply:
ok